src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -459,7 +461,8 @@ private boolean checkFilePermissions(final Path p) {
                 handlers.add(new DashboardsInfoAction(settings, restController, Objects.requireNonNull(evaluator), Objects.requireNonNull(threadPool)));
                 handlers.add(new TenantInfoAction(settings, restController, Objects.requireNonNull(evaluator), Objects.requireNonNull(threadPool),
 				Objects.requireNonNull(cs), Objects.requireNonNull(adminDns), Objects.requireNonNull(cr)));
-
+                handlers.add(new SecurityConfigUpdateAction(settings, restController,Objects.requireNonNull(threadPool), adminDns, configPath, principalExtractor));
+                handlers.add(new SecurityWhoAmIAction(settings ,restController,Objects.requireNonNull(threadPool), adminDns, configPath, principalExtractor));
```

Comment:
Why do these handlers need to be added to replace TransportClient?

Reply:
https://github.com/opensearch-project/security/pull/1638#discussion_r813423437